.github/workflows/aws_deploy.yml

```
@@ -31,15 +37,28 @@ jobs:
         id: set_version
         run: |
           if [[ "$GITHUB_REF" == "refs/heads/main" ]]; then
-            # For main branch, use the latest tag or default to 1.0.0
-            VERSION=$(git describe --tags --abbrev=0 2>/dev/null || echo "1.0.0")
+            # Get the latest version tag
+            LATEST_TAG=$(git describe --tags --abbrev=0 2>/dev/null || echo "v0.0.0")
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)